lib/clusters/thermostat.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const Cluster = require('../Cluster');
+const { ZCLDataTypes } = require('../zclTypes');
+
+const ATTRIBUTES = {
+  localTemperature: { id: 0, type: ZCLDataTypes.int16 },
+  outdoorTemperature: { id: 1, type: ZCLDataTypes.int16 },
+  ocupancy: { id: 2, type: ZCLDataTypes.map8 },
```

Comment:
You need to provide the right values for the `map8` data type, check groups attribute `nameSupport` for example.